Tests/AlicerceTests/Logging/LogTests.swift

```diff
@@ -1,6 +1,6 @@
 //
 //  LogTests.swift
-//  AlicerceTests
+//  Alicerce
```

Comment:
Shouldn't these still be `AlicerceTests`, since they're on the test target?

I know most of them aren't, but perhaps we should fix this the other way around, because when we create a new file and add it to the test target, it assumes the `AlicerceTests` by default.

The fact that most of them are using `Alicerce` is probably because they were created incorrectly by initially being added to the main target 😓

Reply:
I have mixed feelings about this. Xcode uses the target name when creating new files but, on the other hand, with SPM where we don't have such notion of a target, I'm unsure we need to follow this convention. I've taken a look at some popular repositories and found out that they use the main target or no specification at all.

What are your thoughts? Should we append 'Tests'?

Reply:
No strong opinion regarding this. Just mentioned because in a way we're going against Xcode's default behaviour. Nonetheless, that isn't by any means a blocker 😉 

If we decide to use `Alicerce` everywhere, we just have to remember to always fix it whenever we create a new file on the test target.

Thankfully, I think we will be able to enforce this easily using SwiftLint, right @filipe-lemos? 😇